src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/SimpleConfigModel.cs

```diff
@@ -517,6 +606,32 @@ public static SimpleConfigModel FromJObject(JObject source)
             config.Tags = source.ToStringDictionary(StringComparer.OrdinalIgnoreCase, nameof(config.Tags));
             config.PostActionModel = RunnableProjects.PostActionModel.ListFromJArray((JArray)source["PostActions"]);
 
+            // Custom operations at the global level
+            JToken globalCustomConfigData = source[nameof(config.CustomOperations)];
+            if (globalCustomConfigData != null)
+            {
+                config.CustomOperations = CustomFileGlobModel.FromJObject((JObject)globalCustomConfigData, string.Empty);
+            }
+            else
+            {
+                config.CustomOperations = null;
+            }
+
+            // Custom operations for specials
+            IReadOnlyDictionary<string, JToken> allSpecialOpsConfig = source.ToJTokenDictionary(StringComparer.OrdinalIgnoreCase, "SpecialCustomOperations");
```

Comment:
I wonder if we should call specials something more descriptive of their nature - in the config at least. Haven't thought of a good, concise way of expressing "configuration for the actions that will be taken on files whose paths match a pattern", so I've got no suggestion